markdown_lab/core/scraper.py

```
@@ -850,45 +743,107 @@ def scrape_by_links_file(
+                    )
+                    successfully_scraped.append(url)
+                except Exception as e:
+                    failed_urls.append((url, str(e)))
+                    logger.error(f"Error processing URL {url}: {e}")
+                    continue
+
+        # Log results
         logger.info(
             f"Successfully scraped {len(successfully_scraped)}/{len(links)} URLs"
         )
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)